/*
  Software Update

  This API Section holds messages used for installing software somewhere in the vehicle and tracking that process.  For reporting detailed results or logs a Logs Upload Event can be used, which is defined in the API section Device Management. To indicate that the log upload event belongs to the context of an Software update, the value of context in the Logs Upload Event should be set to softwareupdate by convention.

  The version of the OpenAPI document: 2

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.softwareupdate.v2;
import public "device_to_backend_message_meta.proto";
import public "ota_status.proto";

message UpdateStatusEvent {

  DeviceToBackendMessageMeta meta = 3347973;

  Status status = 355610639;

  // Further information on UpdateStatus, intended for error cases
  string message = 418054152;

}